cms/templates/settings.html

```diff
@@ -333,6 +333,9 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                 <li class="field text" id="field-course-overview">
                   <label for="course-overview">${_("Course Overview")}</label>
                   <textarea class="tinymce text-editor" id="course-overview"></textarea>
+                  <label class="sr" for="cm-textarea">
+                    Code Editor
```

Comment:
I'd also suggest a more descriptive label. Since this is being used for HTML editing of the Course Overview, perhaps "HTML Code Editor".